src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/GCHandle.T.cs

```diff
@@ -0,0 +1,114 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.CompilerServices;
+
+namespace System.Runtime.InteropServices
+{
+    /// <summary>
+    /// Represents a strongly-typed opaque, GC handle to a managed object.
+    /// A GC handle is used when an object reference must be reachable from
+    /// unmanaged memory.
```

Comment:
```suggestion
    /// A GC handle is used to work with object references in unmanaged code.
```
"must be reachable from unmanaged memory" sounds like a requirement; not a description what the type is for